editor.planx.uk/src/ui/editor/ComponentTagSelect.test.tsx

```diff
@@ -0,0 +1,114 @@
+import ChecklistComponent from "@planx/components/Checklist/Editor";
+import { within } from "@testing-library/react";
+import { TAG_DISPLAY_VALUES } from "pages/FlowEditor/components/Flow/components/Tag";
+import { useStore } from "pages/FlowEditor/lib/store";
+import React from "react";
+import { DndProvider } from "react-dnd";
+import { HTML5Backend } from "react-dnd-html5-backend";
+import { act } from "react-dom/test-utils";
+import { setup } from "testUtils";
+import { it } from "vitest";
+
+const { setState } = useStore;
+
+const mockUser = {
+  id: 200,
+  firstName: "Testy",
+  lastName: "McTester",
+  email: "test@email.com",
+  teams: [],
+};
+
+describe("Checklist Component for a Platform Admin", () => {
+  beforeEach(() =>
+    act(() =>
+      setState({
+        user: {
+          ...mockUser,
+          isPlatformAdmin: true,
+        },
+      }),
+    ),
+  );
+  it("renders all tags with none selected", async () => {
```

Comment:
nit: Easier to read with more whitespace (imho!)

```suggestion
  );
  
  it("renders all tags with none selected", async () => {
```